src/player.cpp

```diff
@@ -771,7 +767,7 @@ void Player::addStorageValue(const uint32_t key, const int32_t value, const bool
 		storageMap[key] = value;
 
 		if (!isLogin) {
-			int64_t currentFrameTime = OutputMessagePool::getInstance()->getFrameTime();
+			int64_t currentFrameTime = OTSYS_TIME();
```

Comment:
This is intended to make sure "Your questlog has been updated." is only sent once per dispatcher task, checking OTSYS_TIME() is not good enough as it's very likely to have a different value next time it's called in the same dispatcher task.